src/transformers/models/bart/modeling_bart.py

```diff
@@ -582,14 +582,14 @@ def __init_subclass__(self):
             :meth:`transformers.PreTrainedTokenizer.encode` and :meth:`transformers.PreTrainedTokenizer.__call__` for
             details.
 
-            `What are input IDs? <../glossary.html#input-ids>`__
+            `What are decoder input IDs? <../glossary.html#decoder-input-ids>`__
 
             Bart uses the :obj:`eos_token_id` as the starting token for :obj:`decoder_input_ids` generation. If
             :obj:`past_key_values` is used, optionally only the last :obj:`decoder_input_ids` have to be input (see
             :obj:`past_key_values`).
 
             For translation and summarization training, :obj:`decoder_input_ids` should be provided. If no
-            :obj:`decoder_input_ids` is provided, the model will create this tensor by shifting the :obj:`input_ids` to
```

Comment:
this was actually correct - for BART if no `input_ids` **and** no `labels` are provided -> then `decoder_input_ids` are shifted according to the `input_ids` => this corresponds to BART's denoising pre-training